var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -63,15 +64,17 @@ class PyPytest(PythonPackage):
     depends_on("py-pluggy@0.7:", when="@3.7:4.3", type=("build", "run"))
     depends_on("py-pluggy@0.5:0.7", when="@3.6.4:3.6", type=("build", "run"))
     depends_on("py-pluggy@0.5:0.6", when="@:3.6.3", type=("build", "run"))
-    depends_on("py-py@1.8.2:", when="@6:", type=("build", "run"))
+    depends_on("py-py@1.8.2:", when="@6:7.1", type=("build", "run"))
     depends_on("py-py@1.5.0:", when="@3.3:5", type=("build", "run"))
     depends_on("py-py@1.4.33:", when="@3.1.2:3.2.3,3.2.5:3.2", type=("build", "run"))
     depends_on("py-py@1.4.33:1.4", when="@3.2.4", type=("build", "run"))
     depends_on("py-py@1.4.29:", when="@:3.1.1", type=("build", "run"))
```

Comment:
I would move these down to the historical dependencies section

Reply:
Moved down.